velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -36,68 +34,98 @@ namespace fs = std::filesystem;
 namespace fs = std::experimental::filesystem;
 #endif
 
+#include <map>
+#include <vector>
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+
+using TableColumnsMap = std::map<std::string, std::vector<std::string>>;
+
+static const core::SortOrder kAscNullsLast(true, false);
```

Comment:
We can remove these. Masha recently added these to the core so that we can reuse them.
https://github.com/facebookincubator/velox/blob/main/velox/core/PlanNode.h#L80